btr-api/tests/unit/resources/test_submissions.py

```diff
@@ -296,21 +371,26 @@ def test_post_plots_bor_error(app, client, session, jwt, requests_mock):
     ("invalid entity - frozen", True, "ACTIVE", HTTPStatus.FORBIDDEN, ['The business is frozen']),
     ("invalid entity - historical", False, "HISTORICAL", HTTPStatus.FORBIDDEN, ['The business is not active'])
 ])
-def test_post_plots_invalid_entity(app, client, session, jwt, requests_mock, test_name, admin_freeze, state, expected_response, errors):
+def test_post_plots_invalid_entity(app, client, session, jwt, requests_mock, test_name, admin_freeze, state,
```

Comment:
can you add a similar test as this one to validate it returns forbidden when the auth mock does not contain the 'edit' role?

Reply:
there is one with the '_forbidden_auth_response2" as reply for mock